ibis/pyspark/compiler.py

```diff
@@ -402,9 +405,7 @@ def compile_any(t, expr, scope, timecontext, context=None, **kwargs):
 
 
 @compiles(ops.NotAny)
-def compile_notany(
-    t, expr, scope, timecontext, *, context=None, window=None, **kwargs
-):
+def compile_notany(t, expr, scope, timecontext, *, context=None, **kwargs):
```

Comment:
https://github.com/ibis-project/ibis/pull/2409/checks?check_run_id=1178201082#step:4:1284
We removed `window` here and seems tests are failing for this, and `compile_notall`, given the comments here, could you give some suggestions on how we should refactor? @icexelloss

Reply:
This function looks fine and I don't think we need to change this. What exactly failed and how?